app/assets/javascripts/discourse/lib/plugin-api.js

```diff
@@ -1017,6 +1018,24 @@ class PluginApi {
   decoratePluginOutlet(outletName, callback, opts) {
     addPluginOutletDecorator(outletName, callback, opts || {});
   }
+
+  /**
+   * Allows to alter topic list item node
```

Comment:
```suggestion
   * Allows altering the topic title in the topic list, and in the topic view
```

Also would be useful to add the possible values of `topicTitleType`